apps/catalog/catalog/src/environment/environment.ts

```diff
@@ -1,4 +1,3 @@
 import { getEmulatorsConfig } from "@blockframes/utils/emulator-front-setup";
-import { emulators } from "@env";
 
-export const emulatorConfig = getEmulatorsConfig(emulators);
+export const emulatorConfig = getEmulatorsConfig({ auth: false, firestore: false, functions: false });
```

Comment:
```suggestion
export const emulatorConfig = getEmulatorsConfig();
```

and put this `{ auth: false, firestore: false, functions: false }` as default in `getEmulatorsConfig(`

```
export function getEmulatorsConfig(emulators: EnabledEmulators = { auth: false, firestore: false, functions: false }) {
```